### PR TITLE
New package: Google.OneVPN version 1.3.36.292

### PR DESCRIPTION
**Pull request by mezbauddin**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/116725&drop=dogfoodAlpha

### CODE REVIEW

**Comment by mezbauddin** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Hi @mezbauddin,

This seems like another version of this package:
* https://github.com/microsoft/winget-pkgs/pull/116481

File: manifests/g/Googel/OneVPN/1.3.36.292/Googel.OneVPN.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Googel.OneVPN
```
Comment (stephengillie) [inline manifests/g/Googel/OneVPN/1.3.36.292/Googel.OneVPN.installer.yaml:4]:
```suggestion
PackageIdentifier: Google.OneVPN
```
Please update file paths to match this change.

File: manifests/g/Googel/OneVPN/1.3.36.292/Googel.OneVPN.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Googel.OneVPN
```
Comment (stephengillie) [inline manifests/g/Googel/OneVPN/1.3.36.292/Googel.OneVPN.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Google.OneVPN
```
Please update file paths to match this change.

File: manifests/g/Googel/OneVPN/1.3.36.292/Googel.OneVPN.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.5.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: Googel.OneVPN
```
Comment (stephengillie) [inline manifests/g/Googel/OneVPN/1.3.36.292/Googel.OneVPN.yaml:4]:
```suggestion
PackageIdentifier: Google.OneVPN
```
Please update file paths to match this change.

**Comment by mezbauddin** [comment]
> Hi @mezbauddin,
> 
> This seems like another version of this package:
> 
> * [New package: Google.VPNByGoogleOne version 1.7.0.0 #116481](https://github.com/microsoft/winget-pkgs/pull/116481)
> 
> Which PackageIdentifier package part should be used? This one is shorter, but might not fit branding.

Hi @stephengillie  Thank you for the suggestion i am completely new with this, please suggest me the better way to do it .
regarding this package - google one vpn i am not sure if the other request dont have the correct version information please see the ss attached. also please advice why mine is failing on the test
Screenshot 2023-08-15 083606

**Comment by Trenly** [comment]
I don't think I can give an opinion since I get `Unknown Installer Error` anytime I try to install the package

**Comment by mdanish-kh** [comment]
> I don't think I can give an opinion since I get `Unknown Installer Error` anytime I try to install the package

Same issue on my end.

As for the PackageIdentifier: Looking at https://one.google.com/about/vpn the package seems to be marketed with the name `VPN by Google One` everywhere. I believe PR https://github.com/microsoft/winget-pkgs/pull/116481 got it right and we should probably stick to that PackageIdentifier

**Comment by stephengillie** [comment]
>  Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\g\Googel\OneVPN\1.3.36.292', expected 'manifests\g\Google\OneVPN\1.3.36.292'

The file path is still misspelled.

**Pull request closed without merging**